lib/hyrax/controlled_vocabularies/location_decorator.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Hyrax
+  module LocationDecorator
+    # the Location class throws an error that split is undefined since it is an enumerator, but this error is coming from deeply nested dependencies.
+    # This causes 500 errors for works with their Location field filled in.
+    # defining split just for the purpose of getting past this error seemed to work and allow locations to save correctly with no adverse effects.
+    # See https://github.com/scientist-softserv/palni-palci/issues/524 for more details
+    def split(*)
```

Comment:
This didn't seem to have any adverse effects since it is only applying to this one specific class that is only used for this controlled vocab, but please let me know if there is a better way to do this. I searched in Hyrax but there didn't seem to be any differences for any of the files used for this controlled vocab between the newest version and the version PALS is on, so this error is still very mysterious to me. 

The place where split was being called was so deeply nested it was hard to find. But, it seems like just defining it still allows the field to save with the correct location names.

Reply:
how does #524 related to this method/file?

Reply:
@alishaevn good catch i meant to put https://github.com/scientist-softserv/palni-palci/issues/530! not 524